server/db/create_test_data.js

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// this script assumes you've run the create_db script.
+
+'use strict';
+
+var crypto = require('crypto');
+var uuid = require('uuid');
+var program = require('commander');
+
+var config = require('../config');
+var pool = require('./utils').createPool();
+var db = require('./db');
+var log = require('../logger')('db.createTestUser');
+
+// see bottom for initialization
+
+function done() {
+  log.verbose('should now exit; done invoked');
+  process.exit();
+}
+
+function createTestUser(recordCount, cb) {
+  recordCount = recordCount || 25;
```

Comment:
The `25` seems a bit "magic" here (and on lines 112 and 115). We may want to turn it into a const or config value.